FixupOrdering was removed in gdal 2.5.0.

https://github.com/libLAS/libLAS/issues/164

error: no member named 'FixupOrdering' in 'OGRSpatialReference'
--- src/gt_wkt_srs.cpp.orig	2020-05-08 15:45:31.000000000 -0500
+++ src/gt_wkt_srs.cpp	2020-06-08 19:25:08.000000000 -0500
@@ -299,7 +299,6 @@
                 oSRS.SetFromUserInput(pszWKT);
                 oSRS.SetExtension( "PROJCS", "PROJ4",
                                    "+proj=merc +a=6378137 +b=6378137 +lat_ts=0.0 +lon_0=0.0 +x_0=0.0 +y_0=0 +k=1.0 +units=m +nadgrids=@null +wktext  +no_defs" );
-                oSRS.FixupOrdering();
                 CPLFree(pszWKT);
                 pszWKT = NULL;
                 oSRS.exportToWkt(&pszWKT);
@@ -505,7 +504,6 @@
         {
             char	*pszWKT;
             oSRS.morphFromESRI();
-            oSRS.FixupOrdering();
             if( oSRS.exportToWkt( &pszWKT ) == OGRERR_NONE )
                 return pszWKT;
         }
@@ -1107,7 +1105,6 @@
 /* ==================================================================== */
     char	*pszWKT;
 
-    oSRS.FixupOrdering();
 
     if( oSRS.exportToWkt( &pszWKT ) == OGRERR_NONE )
         return pszWKT;
